playground/README.md

```diff
@@ -5,19 +5,24 @@ A web based playground for DefraDB.
 ## Table of Contents
 
 <!--ts-->
+   * [Prerequisites](#prerequisites)
    * [Getting Started](#getting-started)
-   * [Remote mode](#remote-mode)
-   * [Wasm mode](#wasm-mode)
-      * [Local ACP (default)](#local-acp-default)
-      * [SourceHub ACP (requires running SourceHub)](#sourcehub-acp-requires-running-sourcehub)
+      * [Remote mode](#remote-mode)
+      * [Wasm mode](#wasm-mode)
+         * [Local ACP (default)](#local-acp-default)
+         * [SourceHub ACP (requires running SourceHub)](#sourcehub-acp-requires-running-sourcehub)
    * [Building](#building)
 <!--te-->
 
+## Prerequisites
+- Go v1.23 or later
+- Node.js v20 or later
```

Comment:
todo: This appears to be incorrect, v20.0.0 doesnt work for me.  v20.17.0 does, so does the latest lts, and the error message suggested v20.17.* should, and `>=22.9.0`, but v21.* will not.

Might it be worth adding versions explicitly to our package.json?  I do not know what is best practice here https://stackoverflow.com/questions/42805913/how-to-find-what-node-version-a-project-supports